doc/active_docs/account_management_api.json

```diff
@@ -2812,7 +2812,7 @@
                   },
                   "allowed_service_ids[]": {
                     "type": "array",
-                    "description": "IDs of the services that need to be enabled, URL-encoded array. To disable all services, set the value to '[]'. To enable all services, add parameter 'allowed_service_ids[]' with no value to the 'curl' command (can't be done in ActiveDocs)",
+                    "description": "IDs of the services that need to be enabled, URL-encoded array. To disable all services, send `allowed_service_ids[]` with an empty value, e.g. `allowed_service_ids%5B%5D=`. `<allowed_service_ids/>` or `\"allowed_service_ids\":[]` in the response indicate that no services are allowed. To enable all services, send 'allowed_service_ids' with no value to the 'curl' command, e.g. `allowed_service_ids=` (can't be done in ActiveDocs). Missing `<allowed_service_ids>` or `\"allowed_service_ids\":null` indicate that all services are allowed.",
```

Comment:
As we talked on slack, the API Docs are very counter intuitive for this endpoint, for instance, I expect "Send empty value" to set `allowed_service_ids[]` to a value that means `none`. However, checking the box does nothing and the way to actually set allowed services to `none` is to manually inserting any non integer value, which is even more confusing considering the input has a label which says `Add integer item`.

I'm aware on how cumbersome is making any small change in SwaggerUI, so if this can't be properly fixed, at least I think we should mention in the description how to set the `none` case

![image](https://github.com/user-attachments/assets/e74a3096-13fb-41a2-b492-17fcfbac73e1)

Reply:
I agree. I actually updated the description of this parameter, trying to make it more clear, but it is true that unfortunately "send `allowed_service_ids[]` with an empty value" seems impossible through the Swagger UI :grimacing: 

It used to say "set it to `[]`", but I wanted to deprecate this strange way... I can probably add a note "can't be done in ActiveDocs", as I did for the use case of enabling all services.